graft/evm-shared/scripts/lint.sh

```diff
@@ -110,18 +110,25 @@ function test_interface_compliance_nil {
 }
 
 function test_import_testing_only_in_tests {
-  local files=("$@")
   NON_TEST_GO_FILES=$(
-    echo "${files[@]}" | tr ' ' '\n' |
+    echo "${AVALANCHE_FILES[@]}" | tr ' ' '\n' |
       grep -i '\.go$' |
       grep -vi '_test\.go$' |
       grep -v '^./tests/'
   )
 
+  # Detect which repo we're in based on pwd
```

Comment:
if there's a better way to do this, I can totally change this -- the "repositories" are just folders really